src/modules/Categories.ts

```diff
@@ -28,7 +28,7 @@ export const cancelGetCategories = () => getAsync.cancel()
 // ------------------------------------
 
 const initialState: ModelState<Category[]> = {
-	data: null,
+	data: [],
```

Comment:
Why do we need its initial state to be `[]` 🤔  Is `null` easier to check for existence than `[]`?

Reply:
The default value of array should be [] since you can make use of array methods such as push, append, ...